docs/src/main/paradox/release-notes.md

```diff
@@ -1,5 +1,28 @@
 # Release Notes
 
+## 10.0.8
+
+### Ability to express textual content types with missing character set
+
+Akka-http has a strongly typed media type / content type system, and knows at compile time about which media types
+are supposed to express a character set attribute, e.g. `text/plain; charset=UTF-8`. Before this release, akka would
+silently assume UTF-8 for `ContentType` instances of media types with a missing `charset` attribute.
+
+From now on, content types missing a charset can be both parsed and expressed directly, using the new 
+`ContentType.WithMissingCharset` trait/class. 
+
+- If you happen to parse an http request with just `text/plain` as its `Content-Type` header, it'll 
+  show up as an instance of this class.
+- If you need to create such a content type programmatically, use e.g. ```MediaTypes.`text/plain`.withMissingCharset```
+  (scala) or `MediaTypes.TEXT_PLAIN.toContentTypeWithMissingCharset()` (java).
+
+Although this change affects binary compatibility because of the extra introduced method, typical usages should be safe,
+as `ContentType` is a sealed trait within akka-http.
```

Comment:
the additional method does not matter, don't mention it - no one should extend the http model, ever.

The actual potentially breaking issue is the "note to scala users", it is binary compatible though.